repos/system_upgrade/el7toel8/actors/migratesendmail/libraries/library.py

```diff
@@ -0,0 +1,41 @@
+import re
+import ipaddress
+import os
+import shutil
+
+BackupSuffix = '.bak'
+
+# false positives blacklist
+rfp = re.compile(r'(^\s*RIPv6:::1\b)|(@\s+\[IPv6:::1\]\s+>)')
+
+rs = re.compile(r'IPv6:[0-9a-fA-F:]*::[0-9a-fA-F:]*')
+
+try:
+    # Python 2: "unicode" is built-in
+    unicode
+except NameError:
+    unicode = str
+
+def uncompress_ipv6(ipv6):
+    addr = unicode(ipv6.replace('IPv6:', ''))
+    try:
+        addr = 'IPv6:' + ipaddress.ip_address(addr).exploded
+    except ValueError:
+        addr = ipv6
+    return re.sub(r':0([^:])', r':\1', re.sub(r'0+', r'0', addr))
+
+def check_false_positives(f, l):
```

Comment:
Please don't use single letter names for anything that is not a oneliner

Reply:
It should be fixed by @examon commits.